### PR TITLE
[TASK] Add Filesystem and Database excludes as array

### PR DESCRIPTION
**Pull request by Cyb10101**
See issue https://github.com/phpsu/phpsu/issues/69

closes: #69

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,12 +53,14 @@ $config = new \PHPSu\Config\GlobalConfig();
...
-    ->addExclude('*.zip');
+    ->addExclude('*.zip')
+    ->addExcludes(['*.jpg', '*.gif']);
 $config->addSshConnection('hostA', 'ssh://user@localhost:2208');
 $config->addAppInstance('production', 'hostA', '/var/www/')
     ->addDatabase('app', 'mysql://root:password@127.0.0.1:3307/production01db')
     ->addExclude('table1')
-    ->addExclude('table2');
+    ->addExclude('table2')
+    ->addExcludes('table3', 'table4');
```
Comment (Kanti) [inline README.md]:
```suggestion
    ->addExcludes(['table3', 'table4']);
```

**Comment by Kanti** [comment]
@Cyb10101 This PR needs some more tests. For the newly introduced lines.
Then we can merge this.

File: src/Config/Database.php [hunk 2]
```diff
@@ -41,6 +41,12 @@ public function getExcludes(): array
         return $this->excludes;
     }
 
+    public function addExcludes(array $excludes): Database
+    {
+        $this->excludes = array_merge($this->excludes, $excludes);
```
Comment (ChrisB9) [inline src/Config/Database.php]:
```suggestion
        $this->excludes = \array_merge($this->excludes, $excludes);
```

File: src/Config/FileSystem.php [hunk 3]
```diff
@@ -39,6 +39,12 @@ public function getExcludes(): array
         return $this->excludes;
     }
 
+    public function addExcludes(array $excludes): FileSystem
+    {
+        $this->excludes = array_merge($this->excludes, $excludes);
```
Comment (ChrisB9) [inline src/Config/FileSystem.php]:
```suggestion
        $this->excludes = \array_merge($this->excludes, $excludes);
```

File: tests/ControllerTest.php [hunk 4]
```diff
@@ -60,7 +60,7 @@ public function testFilesystemAndDatabase(): void
     public function testExcludeShouldBePresentInRsyncCommand(): void
     {
         $config = new GlobalConfig();
-        $config->addFilesystem('fileadmin', 'fileadmin')->addExclude('*.mp4')->addExclude('*.mp3')->addExclude('*.zip');
+        $config->addFilesystem('fileadmin', 'fileadmin')->addExclude('*.mp4')->addExclude('*.mp3')->addExcludes(['*.zip', '*.rar']);
```
Comment (ChrisB9) [inline tests/ControllerTest.php]: add Exclude should be a test on its own - a test should only test thing at a time...
what do you think @Kanti ?

Comment (Kanti) [reply tests/ControllerTest.php]: I don't have a strong opinion on this.

Comment (Cyb10101) [reply tests/ControllerTest.php]: I do not know it either. If you like you can add a test and just push the commit into the branch.

**Pull request merged**